src/edu/stanford/dlss/was/WasapiResponseParser.java

```diff
@@ -0,0 +1,16 @@
+package edu.stanford.dlss.was;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class WasapiResponseParser {
+
+  public WasapiResponseParser() { }
+
+  public WasapiResponse parse(byte[] jsonData) throws IOException {
+    ObjectMapper objectMapper = new ObjectMapper();
+    WasapiResponse responseObject = objectMapper.readValue(jsonData, WasapiResponse.class);
```

Comment:
great feedback!  I just copied the first example I found.  I have now made this change.